tests/test_sides.py

```diff
@@ -0,0 +1,20 @@
+# Tests for functionality on part sides.
+
+from cycax.cycad import SheetMetal
+from cycax.cycad.location import BACK, BOTTOM, FRONT, LEFT, RIGHT, TOP
+
+sides = (LEFT, RIGHT, TOP, BOTTOM, FRONT, BACK)
+
+
+def test_opposite(tmp_path):
+    """Test that the opposite function of sides work."""
+
+    mypart = SheetMetal(x_size=20, y_size=20, z_size=20, part_no="chunk-of-iron")
+    for side_name in sides:
+        side = getattr(mypart, side_name.lower())
```

Comment:
For the next release `part.get_side(side: str)` would be a nice method.

Reply:
SS179